Make.inc

```diff
@@ -220,6 +220,11 @@ includedir_rel := $(shell $(JULIAHOME)/contrib/relative_path.sh $(bindir) $(incl
 INSTALL_F := $(JULIAHOME)/contrib/install.sh 644
 INSTALL_M := $(JULIAHOME)/contrib/install.sh 755
 
+# BinaryBuilder options
+# TODO: Autodiscover triplet
+USE_BINARYBUILDER_LLVM := 0
+BB_TRIPLET :=
```

Comment:
If this is intended to be at all user-facing, could we use a more descriptive name, perhaps just spelling out BinaryBuilder as in the `USE_BINARYBUILDER_LLVM` variable above?